pkg/resource/resource.go

```diff
@@ -41,6 +41,8 @@ const (
 	FlinkArtifact                    = "Flink artifact"
 	FlinkConnectivityType            = "Flink connectivity type"
 	FlinkComputePool                 = "Flink compute pool"
+	FlinkSavepoint                   = "Flink Savepoint"
+	FlinkDetachedSavepoint           = "Flink Detached Savepoint"
```

Comment:
```suggestion
	FlinkSavepoint                   = "Flink savepoint"
	FlinkDetachedSavepoint           = "Flink detached savepoint"
```
For resource labels, we only capitalize proper names and acronyms.